src/foam/core/FObjectArray.js

```diff
@@ -64,5 +64,39 @@ foam.CLASS({
         return foam.json.parse(value, prop.of, ctx);
       }
     }
+  ],
+
+  methods: [
+    function installInProto(proto) {
+      this.SUPER(proto);
+      var self = this;
+      Object.defineProperty(proto, self.name + '$push', {
+        get: function classGetter() {
+          return function (v) {
+            // Push value
+            this[self.name].push(v);
+            // Force property update
+            this[self.name] = this[self.name];
```

Comment:
Does this work? I thought if the property doesn't change then we don't fire the property change event.

Reply:
That's what I thought too, but when I added this it changed the behaviour to fire every time.